mock-core-configs/etc/mock/templates/rocky-8.tpl

```diff
@@ -133,7 +133,7 @@ enabled=0
 gpgkey=file:///usr/share/distribution-gpg-keys/rocky/RPM-GPG-KEY-rockyofficial
 
 [devel-debug]
-name=Rocky Linux $releasever - Devel WARNING! FOR BUILDROOT AND KOJI USE
+name=Rocky Linux $releasever - Devel - BUILDROOT ONLY!
 mirrorlist=https://mirrors.rockylinux.org/mirrorlist?arch=$basearch&repo=Devel-$releasever-debug
 #baseurl=http://dl.rockylinux.org/pub/rocky/$releasever/Devel/$basearch/debug/tree
 gpgcheck=1
```

Comment:
Why does this exist here? We can't use debuginfo for anything useful and only the `devel` repo has one? @nazunalika?

Reply:
Honestly, speak fast here (today) :-) or delay this to the next release!

Reply:
I'd rather have this removed now and if we want it back, let's have @nazunalika add them all back correctly later.

Reply:
It is enablerepo=0, so I just dropped the "BUILDROOT ONLY" warning which is nowhere else in the debuginfo repos.